packages/vite-plugins/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@ocap/vite-plugins",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Vite plugins for Ocap Kernel development",
+  "homepage": "https://github.com/MetaMask/ocap-kernel/tree/main/packages/vite-plugins#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/ocap-kernel/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/ocap-kernel.git"
+  },
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      }
+    },
+    "./package.json": "./package.json"
+  },
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build": "yarn clean && tsc --project tsconfig.build.json",
```

Comment:
`ts-bridge`'s CommonJS mode disagrees with Vite's types, so we fall back to `tsc`.